hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/keyvalue/KeyValueContainerCheck.java

```diff
@@ -401,6 +401,6 @@ private void handleCorruption(IOException e) {
     String errStr =
         "Corruption detected in container: [" + containerID + "] ";
     String logMessage = errStr + "Exception: [" + e.getMessage() + "]";
-    LOG.error(logMessage);
+    LOG.error(logMessage, e);
```

Comment:
You could clean this up further by removing the two intermediate strings:

```
LOG.error("Corruption detected in container: [{}]. Exception: {}", containerID, e.getMessage, e);
```

Does logging the full stack trace via the last exception parameter not also log the e.getMessage() too?

Reply:
Oops yes it does contain e.getMessage() in the exception object! I'll remove the redundant e.getMessage()! Thanks for the check Stephen!